test/_centos_8.Dockerfile

```diff
@@ -1,5 +1,13 @@
 FROM quay.io/centos/centos:stream8
-RUN yum install -y git initscripts
+RUN yum install -y git systemd
+
+RUN rm -f /lib/systemd/system/multi-user.target.wants/* \
+    /etc/systemd/system/*.wants/* \
+    /lib/systemd/system/local-fs.target.wants/* \
+    /lib/systemd/system/sockets.target.wants/*udev* \
+    /lib/systemd/system/sockets.target.wants/*initctl* \
+    /lib/systemd/system/sysinit.target.wants/systemd-tmpfiles-setup* \
+    /lib/systemd/system/systemd-update-utmp*
```

Comment:
Does `systemd` not detect the container environment and skips conflicting units automatically? Would be closer to real-world testing if systemd is not prevented from starting anything (including its core components) but FTL.

Reply:
> Does systemd not detect the container environment and skips conflicting units automatically? 

I don't know. 
I setup `systemd` in the containers in a way I read somewhere. It made sense to remove unwanted units which are not needed for our own service. However, I see your point: we are not using `docker` here as a way to start a single service but as a proxy for real full-blown systems. I removed the cleaning.

Reply:
Mhh...

> Error: URL https://github.com/pi-hole/ftl/releases/latest/download/pihole-FTL-linux-x86_64 not found

Reply:
Locally, tests pass on `debian_11`